modules/nw-sriov-hwol-supported-devices.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-hardware-offloading.adoc
+
+:_content-type: REFERENCE
+[id="supported_devices_{context}"]
+= Supported devices
+
+Hardware offloading is supported on the following network interface controllers:
+
+.Supported network interface controllers
+[cols="1,2,1,1"]
+|===
+|Manufacturer |Model |Vendor ID | Device ID
+
+|Mellanox
+|MT27800 Family [ConnectX&#8209;5]
+|15b3
+|1017
+
+|Mellanox
```

Comment:
do we support this one `MT28880 Family` too?  I guess we did not test this one, please correct me if I'm wrong @zshi-redhat

Reply:
I have ConnectX-5 Ex card (MT28800 Family) in my environment, which Christopher used for collecting the device information. I was told these two cards (CX-5 and CX-5 Ex) is very similar, but let me double check with FDP team to see if they verify them separately.

Reply:
@zhaozhanqi I believe both cards have been now been confirmed by QE.

Reply:
@zhaozhanqi FDP team verifies OVS HWOL feature for both CX-5 and CX-5 Ex cards, we shall be fine to support them in OCP.

Reply:
ok, Great,  thanks.